viewers/nuxeo-image-viewer.html

```diff
@@ -139,6 +142,15 @@
               type: Boolean,
               value: false,
             },
+
+            /**
+             * If true, the element will show the image in order to fit the viewer.
+             * If false, the element will show the image in real size. 
+             */
+            _fitToViewer: {
+              type: Boolean,
+              value: true,
```

Comment:
we usually don't set the default value in booleans as `true` (see https://www.polymer-project.org/2.0/docs/devguide/properties#configuring-boolean-properties for details), but in this case I think there's no problem (given the explanation in the polymer docs).